### PR TITLE
Expand the 'Math' topic.

### PR DESCRIPTION
**Pull request by ganicke**

### CODE REVIEW

File: common-docs/blocks/math.md [hunk 1]
```diff
@@ -1,36 +1,133 @@
...
+Functions for finding numbers of angles, sides of triangles, and positions on a circle. These
+functions are also used to make information for wave signals.
+
+### Sine
+
+Get the length of the vertical (up or down) side of a right triangle at some angle. But, the
+_sine_ value is the length of the vertical side divided by the length of the longest side,
+it's _hypotenuse_.
+
+What's the sine of 60 degrees? **Math.sin(60)** equals 0.5. The vertical side of a right triangle
```
Comment (pelikhan) [inline common-docs/blocks/math.md]: Isn't JS math using radians?

Comment (ganicke) [inline common-docs/blocks/math.md]: Ah, yes, yes. I'll switch to that.

**Comment by pelikhan** [comment]
See https://github.com/Microsoft/pxt/pull/2292

**Comment by ganicke** [comment]
Would be nice to have Math.PI added too.

**Comment by pelikhan** [comment]
Yes, looking into the JS constants as well.

**Comment by ganicke** [comment]
Killing this PR.

**Pull request closed without merging**